lib/app_server.py

```diff
@@ -245,8 +245,8 @@ def is_correct(run):
 
         test_names = sorted(glob.glob(os.path.join(suite_path, "*.test.lua")))
         test_names = Server.exclude_tests(test_names, test_suite.args.exclude)
-        test_names = sum(map((lambda x: patterned(x, test_suite.args.tests)),
-                             test_names), [])
+        test_names = sum(list(map((lambda x: patterned(x, test_suite.args.tests)),
+                                  test_names)), [])
```

Comment:
Useless change. The existing code works on Python 2 and Python 3.

```python
$ python2
Python 2.7.18 (default, Jan 27 2021, 00:52:11) 
<...>
>>> sum(map((lambda x: [x]), ['a', 'b', 'c']), [])
['a', 'b', 'c']

$ python3
Python 3.9.1 (default, Jan 27 2021, 00:36:29) 
<...>
>>> sum(map((lambda x: [x]), ['a', 'b', 'c']), [])
['a', 'b', 'c']
```

Reverted the commit and verified on test-run too:

```sh
$ git checkout origin/ligurio/gh-15-support-python-3-wip
$ git revert 975335b4d78a94d022deac4378e49e99be66b73f
$ python2 ./test/test-run.py app-tap/msgpack. app/crypto. unit/rtree.
<..all passed..>
$ python3 ./test/test-run.py app-tap/msgpack. app/crypto. unit/rtree.
<..all passed..>
```